files/build_templates/init_cfg.json.j2

```diff
@@ -177,6 +177,12 @@
             "num_dumps": "3"
         }
     },
+{%- if include_p4rt == "y" %}
```

Comment:
Could you add comment about the node-id and how this config is used in the VS p4rt?

Reply:
@baxia-lan Do we need this change in 202405?